ChartIQ/finsemble-react-controls
# 12183 - Updating to react 16.8x

**Resolves issue [12183 ](https://chartiq.kanbanize.com/ctrl_board/18/cards/12183/details)**

**Description of changes**
-Updated React to 16.8.3

**Description of Testing Done**
-Updated seed and Finsemble both seem to be working when manually going through the items in the finsemble-react-controls. No console errors or visual regressions. 

**To Test**
-Open Finsemble seed and make sure to link to the locally cloned finsemble-react-tools. Test each item works the same as before, toolbar, buttons etc.
